python/ql/src/experimental/Security/CWE-522/LDAPInsecureAuth.ql

```diff
@@ -16,5 +16,4 @@ import experimental.semmle.python.security.LDAPInsecureAuth
 
 from LdapInsecureAuthConfig config, DataFlow::PathNode source, DataFlow::PathNode sink
 where config.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "$@ is authenticated insecurely.", sink.getNode(),
-  "This LDAP host"
+select sink.getNode(), source, sink, "This LDAP host is authenticated insecurely."
```

Comment:
We don't even need `.getNode()` here, do we?

(Maybe we need a QL-for-QL query for this, as it seems to be a bit hit-and-miss throughout our queries.)

Reply:
> We don't even need .getNode() here, do we? 

`.getNode()` or not, it produces the same result. So it doesn't matter.  

We might get a performance diff depending on how much magic is pushed down `getLocation()` in `PathNode`/`DataFlow::Node`, but that's it.